trinity/__init__.py

```diff
@@ -5,7 +5,9 @@
 try:
     __version__ = pkg_resources.get_distribution("trinity").version
 except pkg_resources.DistributionNotFound:
-    __version__ = pkg_resources.get_distribution("py-evm").version
+    __version__ = "eth-{0}".format(
```

Comment:
Since this is inside the trinity module, should we just let it crash?

If the concern is confusion from user-reported versions, I still think we might run into problems. If the display on their screen is `eth-v1.2.3` and we ask them the version, it seems totally plausible that they just say "v1.2.3".

Reply:
Merging as-is and punting on a better change which pulls the local git commit and/or reads the version string from `setup_trinity.py`